fdbserver/workloads/SpecialKeySpaceCorrectness.actor.cpp

```diff
@@ -807,6 +807,51 @@ struct SpecialKeySpaceCorrectnessWorkload : TestWorkload {
 				wait(tx->onError(e));
 			}
 		}
+		// test consistencycheck which only used by ConsistencyCheck Workload
+		{
+			try {
+				tx->setOption(FDBTransactionOptions::READ_SYSTEM_KEYS);
+				Optional<Value> val1 = wait(tx->get(fdbShouldConsistencyCheckBeSuspended));
+				state bool ccSuspendSetting =
+				    val1.present() ? BinaryReader::fromStringRef<bool>(val1.get(), Unversioned()) : false;
+				Optional<Value> val2 =
+				    wait(tx->get(SpecialKeySpace::getManagementApiCommandPrefix("consistencycheck")));
+				// Make sure the read result from special key consistency with the system key
+				ASSERT(ccSuspendSetting ? val2.present() : !val2.present());
+				tx->setOption(FDBTransactionOptions::SPECIAL_KEY_SPACE_ENABLE_WRITES);
+				// Make sure by default, consistencycheck is enabled
+				ASSERT(!ccSuspendSetting);
+				// Disable consistencycheck
+				tx->set(SpecialKeySpace::getManagementApiCommandPrefix("consistencycheck"), ValueRef());
+				wait(tx->commit());
+				tx->reset();
+				// Read system key to make sure it is disabled
+				tx->setOption(FDBTransactionOptions::READ_SYSTEM_KEYS);
+				Optional<Value> val3 = wait(tx->get(fdbShouldConsistencyCheckBeSuspended));
```

Comment:
Are we sure we have exclusive ownership of this key? Is it possible that another test could re-enable it before we check?

Reply:
I think we have exclusive ownership of this key.
It is currently not used in any workloads rather than `ConsistencyCheck`.
Even in the `ConsistencyCheck` workload, 
it only reads the key.
At present, I think it's safe to guarantee this.
I added a comment here for this.